frontend/src/worksheets/WorksheetForm.js

```diff
@@ -151,7 +151,7 @@ export default function WorkSheetForm({
                     Munkavégzést igazolja:
                     <Signature name="proofOfEmployment" />
                   </div>
-                  <div className="buttons">
+                  <div className="buttons mt-5">
```

Comment:
Célszerűbb lefele marginolni, nem fölfele. Tehát ebben az esetben az előző elemre egy mb-5-öt rakni.

Reply:
És erre is egy mb-5-öt és akkor a következőre nem kell my.